src/pykeen/experiments/nodepiece/galkin2022_nodepiece_codexl_noancs.yaml

```diff
@@ -0,0 +1,37 @@
+metadata:
```

Comment:
@cthoyt I think configurations outside of the hard-coded naming scheme will not be available through `pykeen experiments reproduce`, right?

Reply:
UPDATE: they can at least be run by `pykeen experiments run`